File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>
int main(void){

int N, min, max;
scanf("%d",&N);
int num[N];

min = 1e6; // 명시적으로 값 할당
max = -1e6; // 명시적으로 값 할당
for (int i = 0; i < N; i++){
scanf("%d",&num[i]);
if (min > num[i])
min = num[i];
if (max < num[i])
max = num[i];
}
printf("%d %d",min,max);
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.*;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int[] z = new int[a];
		int max=-1000000; int min=1000000;
		
		for(int i=0;i<a;i++) {
			z[i]=sc.nextInt();
			 min = z[i]<min?z[i]:min;
			 max = z[i]>max?z[i]:max;
		}
		System.out.print(min+" "+max);
		
	}
	}
